codegen-library.sh

```diff
@@ -27,7 +27,7 @@ fi
 
 export MODULE_NAME=$(go_mod_module_name)
 export GOPATH=$(go_mod_gopath_hack)
-export GOBIN="$(mktemp -d)" # Set GOBIN explicitly as deepcopy-gen is installed by go install.
+export GOBIN=${GOPATH}/bin # Set GOBIN explicitly as deepcopy-gen is installed by go install.
```

Comment:
Just to reflect some context from slack to here, this does mean that we'll pollute the real GOPATH/bin in the $GOPATH case (but not the go mod case). I think that's fine, but I'm biased since I don't use $GOPATH any more ;).

Reply:
As one of the lone GOPATH users, I think it's fine :joy: